tomls/omnibus-arbitrum-mainnet/spot/eth.toml

```diff
@@ -108,4 +108,4 @@ target = ["spotFactory.SpotMarketProxy"]
 fromCall.func = "getMarketOwner"
 fromCall.args = ["<%= extras.synth_eth_market_id %>"]
 func = "setAtomicFixedFee"
-args = ["<%= extras.synth_eth_market_id %>", "<%= parseEther(String(0.01)) %>"]
+args = ["<%= extras.synth_eth_market_id %>", "<%= parseEther(String(0.30)) %>"]
```

Comment:
spot fee change?

Reply:
yeah spot fee change because of the 1 hour staleness